sql-plugin/src/main/spark320/scala/com/nvidia/spark/rapids/RapidsShuffleManager.scala

```diff
@@ -16,8 +16,33 @@
 
 /*** spark-rapids-shim-json-lines
 {"spark": "320"}
+{"spark": "321"}
+{"spark": "321cdh"}
+{"spark": "322"}
+{"spark": "323"}
+{"spark": "324"}
+{"spark": "330"}
+{"spark": "330cdh"}
+{"spark": "330db"}
+{"spark": "331"}
+{"spark": "332"}
+{"spark": "332cdh"}
+{"spark": "332db"}
+{"spark": "333"}
+{"spark": "334"}
+{"spark": "340"}
+{"spark": "341"}
+{"spark": "341db"}
+{"spark": "342"}
+{"spark": "343"}
+{"spark": "350"}
+{"spark": "350db"}
+{"spark": "351"}
+{"spark": "352"}
+{"spark": "353"}
+{"spark": "400"}
 spark-rapids-shim-json-lines ***/
-package com.nvidia.spark.rapids.spark320
+package com.nvidia.spark.rapids.$_spark.version.classifier_
```

Comment:
Are we regressing any functionality in the IDE's ability to understand what's going on with this change?

Reply:
I think the only change for IDE is that the developer needs a warning that the original file should be edited instead of the generated one. I'll add this warning

Reply:
But when editing the original file, I assume the IDE is still totally confused by this package statement and thus doesn't understand what package the file is originally in.  That would be an issue if there were many classes in the package in question, since the IDE would not realize all of those classes don't need to be imported.  But in this specific case, that's not a problem.

Reply:
There is undoubtedly room for confusion. But I think we can mitigate if the need arises. $_spark.version.classifier_ is a legal package name. We could import such files directly and extend interpolation support to anywhere in the code.